### ISSUE TITLE
This sample YAML has wrong description?

### ISSUE THREAD

**Issue by eduidl** [NervanaSystems/distiller#191]
This issue is about https://github.com/NervanaSystems/distiller/blob/master/examples/quantization/quant_aware_train/preact_resnet20_cifar_pact.yaml#L44

```yaml
lr_schedulers:
  training_lr:
    class: MultiStepLR
    milestones: [60, 120]
    gammas: 0.1

```

PyTorch's `MultiStepLR` class requires `gamma` not `gammas`. I suppose `gamma` is corrent, which is wrong?

reference: https://pytorch.org/docs/stable/optim.html#torch.optim.lr_scheduler.MultiStepLR

Regards.

**Comment by nzmora** [comment]
Hi @eduidl,

Thanks! - this is indeed a bug.
@guyjacob - note that this bug appears in two scripts:

- https://github.com/NervanaSystems/distiller/blob/642996cb0df25c1b69e44f66bb83f31aa1cd410c/examples/quantization/quant_aware_train/preact_resnet20_cifar_pact.yaml#L44

- https://github.com/NervanaSystems/distiller/blob/df9a00ce84577cc972ead5a02833d09d656cbd1a/tests/full_flow_tests/preact_resnet20_cifar_pact_test.yaml#L26

**Comment by guyjacob** [comment]
Note that the bad argument was being ignored without any warning. But, by chance, the default value for the proper argument `gamma` was also 0.1, so the actual training ran as expected. This silent ignoring is addressed in #204.

@eduidl Thanks again for catching this.

**Comment by eduidl** [comment]
I have never thought validation of YAM would be implemented from my issue.
I am glad to have created this issue! 

Thank you, too
